README.md

```diff
@@ -107,51 +107,44 @@ keys_gen.sh can be used to generate random passwords for each service.env.
 
 ***NOTE***: Changing the database password after the H2 database has been created requires the database file to be deleted and recreated.
```

Comment:
Keystores containing private keys can be stored in the database - `<fdo-pri-src>/component-sample/demo/{component}/app-data/emdb.mv.db`
as well as in the mounted file system. During runtime, the deployer can decide the mode of Keystore IO by activating the required worker class.

We are not using H2 anymore. Can you remove this section ?